src/pages/student_fees_list/FeesList.tsx

```diff
@@ -0,0 +1,277 @@
+    <StudentsContainer>
+      <InformationStudent>
+        <TitleContainer>
+          <img
+            src={BackIcon}
+            alt="back-icon"
+            onClick={() => navigate(-1)}
+            style={{ cursor: "pointer" }}
+          />
+          <Title>LISTADO DE CUOTAS</Title>
+        </TitleContainer>
+        <SubTitle>
+          <img src={StudentIcon} alt="student-icon" width={20} height={20} />
+          {student?.name} {student?.lastname}
+        </SubTitle>
+        <SubTitle>Cantidad de días: {studentDetails?.numberOfDays}</SubTitle>
+        <SubTitle>Día de pago: {studentDetails?.paymentDay}</SubTitle>
+      </InformationStudent>
+      <FiltersContainer>
+        <LeftContainer>
+          <Filter
+            placeholder="Filtrar por mes"
+            options={[
+              { label: "Enero", value: "JANUARY" },
+              { label: "Febrero", value: "FEBRUARY" },
+              { label: "Marzo", value: "MARCH" },
+              { label: "Abril", value: "APRIL" },
+              { label: "Mayo", value: "MAY" },
+              { label: "Junio", value: "JUNE" },
+              { label: "Julio", value: "JULY" },
+              { label: "Agosto", value: "AUGUST" },
+              { label: "Septiembre", value: "SEPTEMBER" },
+              { label: "Octubre", value: "OCTOBER" },
+              { label: "Noviembre", value: "NOVEMBER" },
+              { label: "Diciembre", value: "DECEMBER" },
+            ]}
+            value={monthFilter}
+            onSelect={setMonthFilter}
+          />
+          <DateFilter
+            value={expirationDateFilter}
+            onChange={setExpirationDateFilter}
+          />
+
+          <Filter
+            placeholder="Filtrar por estado"
+            options={[
+              { label: "Pendiente", value: "PENDING" },
+              { label: "Vencida", value: "OUT_OF_TIME" },
+              { label: "Pagado", value: "PAYED" },
+              { label: "Pagado vencido", value: "PAYED_OUT_OF_TIME" },
+            ]}
+            value={statusFilter}
+            onSelect={setStatusFilter}
+          />
+
+          <Button
+            variant="primary"
+            size="small"
+            onClick={() => {
+              setStatusFilter("");
+              setExpirationDateFilter(null);
+              setMonthFilter("");
+            }}
+          >
+            Limpiar filtros
+          </Button>
+        </LeftContainer>
+        <RightContainer>
+          <Button
+            variant="primary"
+            size="medium"
+            icon={<img src={AddIcon} alt="Add Icon" />}
+            //onClick={() => navigate("/")} Para cuando este la ruta a la nueva cuota.
+          >
+            Nueva cuota
+          </Button>
+        </RightContainer>
+      </FiltersContainer>
+      {modalType === "pay" && (
+        <ConfirmDialog
+          message="¿Estás seguro de realizar este pago?"
+          onConfirm={handleConfirmPay}
+          onCancel={handleCancelPay}
```

Comment:
Este metodo por lo que veo solo setea el modalType en `null`, por lo que podriamos hacerlo directamente aca para no tener que definir un metodo


```suggestion
          onCancel={() => setModalType(null)}
```